### PR TITLE
Jade/110 updating vid conferencing link gen

### PR DESCRIPTION
**Pull request by jade-law**
Updated video conferencing link generation to support new Twilio integration. Link new appears on page as well

### CODE REVIEW

File: server/mongodb/MeetingDocument.ts [hunk 1]
```diff
@@ -21,11 +21,13 @@ const MeetingSchema = new Schema(
...
     },
-    meetingId: {
-      type: Number,
-    },
     meetingLink: {
       type: String,
+      required: true,
+    },
+    meetingName: {
+      type: String,
```
Comment (Navbryce) [inline server/mongodb/MeetingDocument.ts:29]: unique: true for meeting name

File: server/mongodb/actions/MeetingManager.ts [hunk 2]
```diff
@@ -132,30 +133,18 @@ export function docToMeetingCore(object: { [key: string]: any }): MeetingCore {
...
-      meeting.meetingId = json.meetingId;
-      meeting.meetingLink = json.viewerLink;
-    });
+export function genConferenceLinks(
+  meeting: Meeting,
+  meetingDateTime: DateTime
+) {
+  const meetingDateFormat = Object.assign(DateTime.DATE_SHORT);
+  const meetingTimeFormat = Object.assign(DateTime.TIME_24_SIMPLE);
+  meeting.meetingName = `${meeting.nonprofit}-${meetingDateTime.toLocaleString(
```
Comment (Navbryce) [inline server/mongodb/actions/MeetingManager.ts:146]: Does this use the nonprofit name or id?

**Review by Navbryce — CHANGES_REQUESTED** [review]
Looks good! A couple small tweaks

File: server/mongodb/actions/MeetingManager.ts (hunk 2 above)
Comment (jade-law) [reply server/mongodb/actions/MeetingManager.ts:146]: Takes the nonprofit name, should I change to id?

**Review by Navbryce — APPROVED** [review]
LGTM

**Pull request merged**